### DISCUSSION TITLE
How to provide single set of login credentials with all E2E spec file while executing it in parallel mode?

### DISCUSSION THREAD

**Discussion by abhikhanzode** [cypress-io/cypress#27807] [General]
Hello...!

We are attempting to run E2E spec files in parallel on multiple machines using Cypress Cloud by integrating with Azure CI. We are currently encountering a challenge with user login details, as there is only a single set of login credentials needed to provide for all E2E spec files. This is causing conflicts during parallel test execution. Unfortunately, we cannot provide separate user login details for each E2E spec file. 

How can we resolve this issue? Could someone provide guidance on this problem?

Thank You,

**Comment by mohdbilal2000** [comment]
@abhikhanzode did you tried crewting custom command in command.js file?
You can upload these files on cloud to give you one login privileges for testing.

**Reply by abhikhanzode** [reply]
Thanks for asking. 

Yes, it has already been implemented in our framework. However, the problem arises when running two cases in parallel using the same login credentials. Both cases perform the same action on the same page, such as creating a record and saving it. This leads to a conflict in which the application throws an error. However, when executed individually or sequentially, both cases run fine.

**Reply by mohdbilal2000** [reply]
well i recently tried creating login with sessions, are you working with sessions or using any other approach for the one login?
